google-cloud-logging/src/main/java/com/google/cloud/logging/v2/stub/ConfigServiceV2StubSettings.java

```diff
@@ -536,12 +629,12 @@ private static Builder initDefaults(Builder builder) {
 
       builder
           .updateSinkSettings()
-          .setRetryableCodes(RETRYABLE_CODE_DEFINITIONS.get("idempotent"))
+          .setRetryableCodes(RETRYABLE_CODE_DEFINITIONS.get("non_idempotent"))
```

Comment:
again, do we expect to switch here and below?

Reply:
This was a bug in the gapic v2 config and was fixed upstream.